packages/location-field/src/stories/Desktop.story.tsx

```diff
@@ -251,3 +251,35 @@ export const WithUserSettings = (): JSX.Element => (
     userLocationsAndRecentPlaces={userLocationsAndRecentPlaces}
   />
 );
+
+export const WithOtherFirst = (): JSX.Element => {
+  const [otherControl, setOtherControl] = useState(false);
+  return (
+    <>
+      <label htmlFor="other-input">
```

Comment:
Could you make Storybook create the controls using `argTypes` instead? (Storybook will add that to its "Controls" tab - I should have mentioned that in my previous review but somehow thought you knew) Do that for this story only. Refer to e.g.https://github.com/opentripplanner/otp-ui/blob/f75dd895f565834f3cd0685120c1d8ef640b9469/packages/route-viewer-overlay/src/RouteViewerOverlay.story.tsx#L102  for an example.